master/internal/api/handler.go

```diff
@@ -9,7 +9,13 @@ import (
 	"github.com/pkg/errors"
 )
 
-var upgrader = websocket.Upgrader{}
+func bypasscheckorigin(r *http.Request) bool {
+	return true
+}
+
+var upgrader = websocket.Upgrader{
+	CheckOrigin: bypasscheckorigin,
+}
```

Comment:
I didn't know about this header.

This is a fine solution for this draft PR but we'll need to fix the server to handle this better in the near future.